src/OrchardCore.Modules/OrchardCore.Contents/Views/_ViewImports.cshtml

```diff
@@ -12,7 +12,6 @@
 @using OrchardCore.ContentManagement.Metadata.Settings
 @using OrchardCore.Contents.Workflows.Activities
 @using OrchardCore.Contents.Workflows.ViewModels
-@using OrchardCore.Deployment
```

Comment:
Why remove this and add it to the other templates?

Reply:
Because else I get this error when trying to build these templates : 

```
'CommonPermissions' is an ambiguous reference between 'OrchardCore.Contents.CommonPermissions' and 'OrchardCore.Deployment.CommonPermissions' [E: \ Repositories \ OrchardCore \ src \ OrchardCore.Modules \ OrchardCore.Contents \ OrchardCore.Contents.csproj]
```